mobile/lib/services/rest_api.dart

```diff
@@ -283,6 +284,79 @@ class AirqoApiClient {
     return airQualityReadings.removeInvalidData();
   }
 
+  Future<List<FavouritePlace>> fetchFavoritePlaces(String userId) async {
+    final favoritePlaces = <FavouritePlace>[];
+    final queryParams = <String, String>{}
+      ..putIfAbsent('tenant', () => 'airqo');
+
+    if (userId.isEmpty) {
+      return [];
+    }
+
+    try {
+      final body = await _performGetRequest(
+        queryParams,
+        "${AirQoUrls.favourites}/users/$userId",
+        apiService: ApiService.auth,
+      );
+
+      for (final favorite in body['favorites'] as List<dynamic>) {
+        try {
+          favoritePlaces.add(
+            FavouritePlace.fromJson(
+              favorite as Map<String, dynamic>,
+            ),
+          );
+        } catch (exception, stackTrace) {
+          await logException(
+            exception,
+            stackTrace,
+          );
+        }
+      }
+    } catch (exception, stackTrace) {
+      await logException(
+        exception,
+        stackTrace,
+      );
+    }
+
+    return favoritePlaces;
+  }
+
+  Future<bool> syncFavouritePlaces(List<FavouritePlace> favorites,
+      {bool clear = false},) async {
```

Comment:
@BenjaminSsempala we are missing a comma.

Reply:
@NoahNsimbe I fixed this . Thanks